USAGE

```diff
@@ -51,6 +51,42 @@ macos                                    When this profile is used, mrds_flow wi
 
 cbrain                                   When this profile is used, Nextflow will copy all the output files in publishDir and not use symlinks.
 
+OUTPUT
+
+The output files have the following format:
+
+`<sid>__MRDS_<method>_<modsel>_<metric>.nii.gz`,
```

Comment:
i don't understand here because you wrote in the process Fit_MRDS 
${sid}__MRDS_Diff_V1_COMP_SIZE.nii.gz

Is it always Diff ?

Reply:
Yeah, it is always Diff

Reply:
Well it's not always Diff, but for our purposes we always use Diff

Reply:
If we cannot change this method to Fixed or Equal, just mention that the "Diff" method from Ricardo's paper is used, where all tensors eigenvalues are estimated and are allowed to be different.